CHANGELOG.md

```diff
@@ -62,6 +62,11 @@ If you use `recombine` operator you will observe that logs which do not match re
 
 ### Released 2024-02-20
 
+### Known issue
+
+Behavior of [recombine][recombine] operator has been changed. Now it recombines also partial logs, for details please see [pull request][opentelemetry-collector-contrib#30797].
```

Comment:
Does this change prevent us from upgrading the otel collector version in the helm chart?

Reply:
Yes